es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze:', address);
```

Comment:
Looks like the output doesn't quite match:

---

![image](https://user-images.githubusercontent.com/16143771/127082330-d41b682e-9bfa-4211-b80f-b9c8d7ae3c69.png)

---

![image](https://user-images.githubusercontent.com/16143771/127082273-8094b26c-9277-4bfc-9bc6-199f1c5dbeaf.png)